README.md

```diff
@@ -2,7 +2,15 @@
 
 Speed up file transfers with the Hub.
 
-Supports download and upload.
+# DISCLAIMER
+
+This library is a power user tool, to go beyond `~500MB/s` on very high bandwidth
+network, where Python cannot cap out the available bandwidth.
+
+This is *not* meant to be a general usability tool.
+It purposefully lacks progressbars and comes generally as-is.
```

Comment:
Because it can slow things down, and I don't want to spend time investigating why, nor maintaining it.

If it was meant for general use, I'd do it. But it's not (at least I don't).